<ui version="4.0" >
 <class>ProjectPage</class>
 <widget class="QWidget" name="ProjectPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Define Project</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This wizard will help you to create a &lt;span style=" font-weight:600;">XINX&lt;/span> project and define some important variables.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In &lt;span style=" font-weight:600;">XINX&lt;/span>, a project is used to use the &lt;span style=" font-style:italic;">file research &lt;/span>in a project, use a &lt;span style=" font-style:italic;">revision control system&lt;/span>, or use a &lt;span style=" font-style:italic;">project prefix&lt;/span> in derivated modules.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">A project can define web services too.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QRadioButton" name="m_derivatedRadio" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In a derivated project, the file prefix is used to check if a file is derivated or standard.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">A derivated file, he begins bye the prefix and an underscore like '&lt;span style=" font-style:italic;">&amp;lt;prefix&amp;gt;_&amp;lt;filename&amp;gt;.&amp;lt;ext&amp;gt;&lt;/span>'.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If you save a standard file, &lt;span style=" font-weight:600;">XINX&lt;/span> propose you to derive the file, and if possible to add the file in the &lt;span style=" font-style:italic;">revision control system&lt;/span>.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>&amp;Derivated from standard</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QRadioButton" name="m_standardRadio" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If the project is full standard, the project don't ask you to save the file with a specifique prefix.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>Change the &amp;standard</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Name of the project :</string>
     </property>
     <property name="buddy" >
      <cstring>m_projectNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="m_projectNameEdit" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Name of the project showed in the title bar of &lt;span style=" font-weight:600;">XINX&lt;/span>, the project name is appended to the application server directory and showed the project directory&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&amp;Application Server Directory :</string>
     </property>
     <property name="buddy" >
      <cstring>m_ASPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="DirectoryEdit" name="m_ASPathEdit" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">The &lt;span style=" font-style:italic;">application serveur directory&lt;/span> is used to fill the &lt;span style=" font-style:italic;">applcation server log directory&lt;/span>, and the &lt;span style=" font-style:italic;">project directory&lt;/span>. The default value for this field is the &lt;span style=" font-style:italic;">application server directory&lt;/span> from the settings dialog.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Application Server &amp;Log Directory :</string>
     </property>
     <property name="buddy" >
      <cstring>m_logPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="DirectoryEdit" name="m_logPathEdit" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the server log directory, used to list presentation files in the &lt;span style=" font-style:italic;">XML presentation View&lt;/span>.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>&amp;Project Directory :</string>
     </property>
     <property name="buddy" >
      <cstring>m_projectPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="DirectoryEdit" name="m_projectPathEdit" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the project directory, generally where is stored the &lt;span style=" font-style:italic;">configuration.xml&lt;/span> file.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the racine where the &lt;span style=" font-style:italic;">project file view&lt;/span> start.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" >
    <widget class="QToolButton" name="m_ASPathBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../application.qrc" >:/images/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QToolButton" name="m_logPathBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../application.qrc" >:/images/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QToolButton" name="m_projectPathBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../application.qrc" >:/images/folder.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEdit</class>
   <extends>QLineEdit</extends>
   <header>directoryedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../application.qrc" />
 </resources>
 <connections/>
</ui>
